### ISSUE TITLE
[4.0] Changelog Bugs with diacritics

### ISSUE THREAD

**Issue by klucon** [joomla/joomla-cms#29262]
### Steps to reproduce the issue
1) Download test modul from https://build.klucon.cz/mod_kluconholiday_1_2_0.zip (only Czech lang, but it is not problem)
2) Go to Administrator -> System -> Extensions and search word "klucon" and open it
3) Click to version number and compare with the file changelog_mod_kluconholiday.xml
4) In the file is only 3 lines, but in the changelog is 6 lines
5) For the all files I was use UTF-8 without BOM

### System information (as much as possible)

I was tested on last Night Build from citate: "These packages were last built: Friday, 29 May 2020 02:00:24 UTC"

**Comment by ReLater** [comment]
Confirmed.

I don't know if it should work in a different way but I would enclose the lines with CDATA comments

```
<fix>
		<item><![CDATA[
			Oprava položky u data 15-12
		]]></item>
</fix>
```

![30-05-_2020_00-31-54](https://user-images.githubusercontent.com/20780646/83310656-0deafd00-a20d-11ea-8f78-9f6240db8b20.png)

**Comment by klucon** [comment]
Thank you ReLater,

when I used CDATA, it is OK.

**Comment by ReLater** [comment]
Yes, but I couldn't find a reason why it doesn't work without CDATA. Tried a lot to enforce UTF-8 while parsing... checked encoding of your file... added `<?xml version="1.0" encoding="UTF-8"?>` ...

**Comment by richard67** [comment]
Maybe encoding of the webserver itself is still not utf-8 (just to be sure)?

**Comment by richard67** [comment]
If all is set up for utf-8 it should not need the CDATA in my opinion, but I might be wrong.

**Comment by ReLater** [comment]
I've tested it also with my custom site (and a custom changelogurl on it) that runs with utf-8 for sure.

Maybe the httpRequest (I don't know how it's called correctly) gets the wrong encoding ? The wrapped lines happen very early in the PHP parser logic.

I've found a very old comment yesterday https://www.php.net/manual/en/function.xml-parser-create.php#100099

> The original parser sometimes wrapped the text before the first diacritics appearance.

**Comment by richard67** [comment]
@ReLater The comment after that one reads also interesting, encoding ... even if 15 years old it could still be relevant.

**Comment by ReLater** [comment]
I didn't try with UTF-8 plus BOM because that wouldn't be practicable for devs. Perhaps @klucon wants to try? Just for the sake of clarity.

**Comment by klucon** [comment]
Hello, 
now I have converted all the files to UTF-8 with BOM and the same problem.

Here is a UTF-8 ZIP file with BOM:

https://build.klucon.cz/mod_kluconholiday_with_bom.zip

If I use CDATA, everything is alright

> Maybe encoding of the webserver itself is still not utf-8 (just to be sure)?

I am testing on the XAMPP (compiled Jun 5th 2019) and Windows 10
Apache/2.4.41 (Win64) OpenSSL/1.1.1c PHP/7.3.12 



> Yes, but I couldn't find a reason why it doesn't work without CDATA. Tried a lot to enforce UTF-8 while parsing... checked encoding of your file... added `<?xml version="1.0" encoding="UTF-8"?>` 

when I added `<? xml version = "1.0" encoding = "UTF-8"?>,` no change took effect

**Comment by ReLater** [comment]
@klucon 
Thank you for testing!

**Comment by brianteeman** [comment]
tested with and without bom - no change :(
